src/modules/licences/routes.js

```diff
@@ -126,5 +126,24 @@ module.exports = {
         }
       }
     }
+  },
+
+  postLicenceName: {
+    method: 'POST',
+    path: `${pathPrefix}{documentId}/rename`,
+    handler: controller.postLicenceName,
+    config: {
+      description: 'Sets name for a licence',
+      validate: {
+        params: {
+          documentId: Joi.string().guid().required()
+        },
+        payload: {
+          documentName: Joi.string().required().allow(''),
+          rename: Joi.boolean().required(),
```

Comment:
As above, not sure the rename param is needed

Reply:
done